Gemfile

```diff
@@ -31,9 +33,12 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+gem 'simplecov'
```

Comment:
Recommend you add `coverage` to your .gitignore if you're going to use 'simplecov'